shellwords.cabal

```diff
@@ -57,6 +57,8 @@ test-suite hspec
   default-extensions:
       NoImplicitPrelude
   ghc-options: -Weverything -Wno-all-missed-specialisations -Wno-missed-specialisations -Wno-missing-exported-signatures -Wno-missing-import-lists -Wno-missing-local-signatures -Wno-monomorphism-restriction -Wno-safe -Wno-unsafe -threaded -rtsopts -with-rtsopts=-N
+  build-tool-depends:
+      hspec-discover:hspec-discover
```

Comment:
If you add a file `.gitattributes` with
```
*.cabal linguist-generated=true
```
then GitHub indicates in code reviews that this is a non-essential change.

Reply:
Wish there were a way to do this centrally. Not psyched about having to drop this file across a dozens of projects...